avc/src/start_detector/start_detector.cpp

```diff
@@ -2,7 +2,9 @@
 #include <pluginlib/class_list_macros.h>
 #include <cv_bridge/cv_bridge.h>
 #include <std_msgs/Bool.h>
+#include <sensor_msgs/Image.h>
 #include <numeric>
+#include <opencv2/highgui.hpp>
```

Comment:
both of these new includes are already included in the header so they don't do anything